BeerOverflowWindowsApp/ManualBarRating.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Drawing.Drawing2D;
+using System.Drawing.Imaging;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace BeerOverflowWindowsApp
+{
+    public partial class ManualBarRating : Control
+    {
+        List<Rectangle> beerGlassList;
+        [DesignerSerializationVisibility(DesignerSerializationVisibility.Visible)]
+        public int ImageSize{get;set;}
+        //int ImageSize;
+
+        public ManualBarRating()
+        {
+            this.DoubleBuffered = true;
+            InitializeComponent();
+            this.ImageSize = 100;
+            this.MinimumSize = new Size(ImageSize * 6, ImageSize);
+            this.beerGlassList = new List<Rectangle>(6);
+            for(int i = 0; i < 6; i++)
+            {
+                beerGlassList.Add(new Rectangle(i * ImageSize, 0, ImageSize, ImageSize));
+            }
+            Refresh();
+        }
+
+        protected override void OnPaint(PaintEventArgs pe)
+        {
+
```

Comment:
Unnecessary gap.

Reply:
Will remove all unnecessary gaps after I finish this class completely.